src/Searchable.php

```diff
@@ -469,7 +497,7 @@ protected function updateDependentClasses()
 
                 foreach ($list as $object) {
                     if ($object instanceof DataObject && $object->hasExtension(Searchable::class)) {
-                        if (!$object->hasField('ShowInSearch') || $object->ShowInSearch) {
+                        if (!$object->hasField('ShowInSearch') || $object->getField('ShowInSearch')) {
```

Comment:
Minor point (and the logic predates this work) but there are a ton of `ShowInSearch` checks. Perhaps pull that into a static method/helper somewhere?

Reply:
Might leave this for later, seperate from the upgrade work.